yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -609,15 +611,15 @@ export class L1Publisher {
     //        This means that we can avoid the simulation issues in later checks.
     //        By simulation issue, I mean the fact that the block.timestamp is equal to the last block, not the next, which
     //        make time consistency checks break.
```

Comment:
Does this comment still apply?

Reply:
Should not be the case. Also, we should be able to get rid of the `validateBlockForSubmission` here if the send can actually simulate it.

Reply:
ah I wasn't sure on that, I think I mentioned removing `validateHeader` at some meeting and we decided to keep it for now? happy to remove it

Reply:
I take it this will be addressed in another PR then, right?